### PR TITLE
Components

### PR DESCRIPTION
**Pull request by kensteph**
In this PR I : 
- Remove all the boilerplate files.
- Create a Calculator component.
- Fix linter errors.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Munsa1) [inline src/App.js:6]: - [ ] `App.js` should be a class based component, kindly fix this

File: src/components/Calculator/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from '../Button/Button';
+import './calculator.css';
+
+const Calculator = () => {
+  const operatorsBtnColor = '#f4b848';
+  return (
+    <div className="calculator-wrapper">
+      <div className="display">0</div>
```
Comment (Munsa1) [inline src/components/Calculator/Calculator.js:9]: - [ ] `Calculator.js` should also be a class based component, kindly implement this as well

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @kensteph, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Removed all boilerplate code :heavy_check_mark: 
Created calculator UI :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (kensteph) [reply src/App.js:6]: Hi @Munsa1 ! Thank you I will fix it.

File: src/components/Calculator/Calculator.js (hunk 2 above)
Comment (kensteph) [reply src/components/Calculator/Calculator.js:9]: Hi @Munsa1 ! Thank you for your review. But the requirements are very clear : 
image

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @kensteph , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Boilerplate code deleted
:heavy_check_mark: Component folder created with Calculator.js component inside
:heavy_check_mark:  Project requirements structured respected

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 
## NOTE
There was a dicrepancy between the code review guidelines and Project guidelines provided to the students.
- Code review guideline talks of class based components
- Project guidlines provided to students talk of functional components.
As such I have followed the project guidelines provided to students and that is my reason  for approving this milestone
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**